sites/all/themes/wille/sass/component/_carousel.scss

```diff
@@ -11,13 +11,29 @@
 
 $carousel-outer-padding: 0 30px;
 
+.ding-carousel__wrapper {
+  position: relative;
+  z-index: 1;
+}
+
+.ding-carousel__background {
+  position: absolute;
+  z-index: -1;
+  top: 0;
+  bottom: 0;
+  left: -9999px;
+  right: -9999px;
+}
+
 .ding-carousel {
+  color: #fff;
+
   ul {
     display: flex;
     vertical-align: top;
     position: relative;
     height: auto;
-    margin: 0 0 25px 0;
+    margin: 0 -$mobile-margin $mobile-margin 0;
```

Comment:
Den her er tricky. Min interne parser læser det som `top: 0, right: -$mobile-margin, bottom: $mobile-margin, left: 0` men CCs warning antyder at den bliver parset som `top: (0 - $mobile-margin), sides: $mobile-margin, bottom: 0`... Ingen af dem giver rigtig mening for mig?